### PR TITLE
add Play Core Software Development Kit Terms of Service template

### PR DESCRIPTION
**Pull request by 5hyn3**
Add Play Core Software Development Kit Terms of Service template.

Here's an example of using this template.

7ce6ed0adef8088eb482f169fa0c6d0b
c4bda51e22b4ac8a94f533fa2483a32c

### CODE REVIEW

File: plugin/src/test/resources/yaml/licenses.yml [hunk 1]
```diff
@@ -59,3 +59,8 @@
   license: Amazon Software License
   licenseUrl: https://aws.amazon.com/asl/
   url: https://github.com/aws/aws-sdk-android
+- artifact: com.google.android.play:core:+
+  name: core
+  copyrightHolder: Google Inc.
+  license: Play Core Software Development Kit Terms of Service
+  licenseUrl: https://developer.android.com/guide/playcore/license
```
Comment (k4zy) [inline plugin/src/test/resources/yaml/licenses.yml:66]: Please add end of line.

Comment (5hyn3) [reply plugin/src/test/resources/yaml/licenses.yml]: I fixed it.

**Review by k4zy — APPROVED** [review]

**Comment by k4zy** [comment]
Thanks!

**Comment by k4zy** [comment]
@5hyn3 🚀 [version 1.2.5](https://plugins.gradle.org/plugin/com.cookpad.android.plugin.license-tools)

**Pull request merged**